src/rgw/rgw_sal_motr.cc

```diff
@@ -1034,11 +1035,37 @@ int MotrBucket::list(const DoutPrefixProvider *dpp, ListParams& params, int max,
       rgw_bucket_dir_entry ent;
       auto iter = vals[i].cbegin();
       ent.decode(iter);
+      std::string key_name = ent.key.name + "/null";
+      if(keys[i] == key_name)
+      { 
+        // read data for null_index entry - (obj1/null)
+        bufferlist bl_null;
+        ret_code = this->store->do_idx_op_by_name(bucket_index_iname,
+                                    M0_IC_GET, key_name, bl_null);
+        ldpp_dout(dpp, 2) << "**** null key found for object -"<< keys[i] << dendl;
```

Comment:
We can have one debug log instead of two. For All the debug logs use log level 20 and above.
Use `__func__` in all the logs. Print correct rc.
use log level 0 for all the error logs and very important logs.

Reply:
Sure, done